### ISSUE TITLE
How to prepare my own training data

### ISSUE THREAD

**Issue by ArtlyStyles** [balancap/SSD-Tensorflow#390]
Hi, I would like use this to detect objects for autonomous cars. I have collected my own images. I can also label my images by drawing the boundingbox of cars, person, poles, signs, bikes, etc. But in what format should I write this label data?

**Comment by ArtlyStyles** [comment]
Anyone can help?

**Comment by zhouzy-creator** [comment]
Use the tool below
https://github.com/tzutalin/labelImg
